app/models/user.rb

```diff
@@ -561,15 +559,7 @@ def any_kind_of_delegate?
   end
 
   def trainee_delegate?
-    delegate_status == "trainee_delegate"
-  end
-
-  def staff_delegate?
-    any_kind_of_delegate? && !trainee_delegate?
-  end
-
-  def full_delegate?
```

Comment:
Same here, where did it go?

Reply:
This method is not used anywhere, hence removed it.